src/Serilog.Extensions.Logging.File/Serilog/Extensions/Logging/File/FileLoggingConfiguration.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Serilog.Extensions.Logging.File
+{
+    /// <summary>
+    /// Configuration for the Serilog file logging.
+    /// </summary>
+    public class FileLoggingConfiguration
```

Comment:
Can we get away with making this `internal`?

Reply:
Sure, no problem with that, it was just a reflex since I usually make everything public. Since this type is only used internally for binding the JSON configuration, there’s no problem with that here. I’ll update the PR.